hw/misc/hermes.c

```diff
@@ -1019,17 +1040,24 @@ static void pci_hermes_realize(PCIDevice *pdev, Error **errp)
     hermes_init_msix(hermes, errp);
 
     hermes->stopping = false;
-    hermes_bar2_init_dir_reg(hermes, &hermes->bar2.h2c);
-    hermes_bar2_init_dir_reg(hermes, &hermes->bar2.c2h);
+
+    for(size_t i = 0; i < H2C_CHANNELS; i++)
```

Comment:
nit: don't define the iteration variable on the for loop

Reply:
Do you want the `i` defined on the line above. Why would that change anything?

Reply:
Not at the line above, but at the beginning of the function. It's the QEMU's coding style.

QEMU also provides a scripts/checkpatch.pl script that can verify for style violations. I'd recommended that you configure your git pre-commit hook to [run it automatically](http://blog.vmsplice.net/2011/03/how-to-automatically-run-checkpatchpl.html).